### PR TITLE
Fixed issue when include groups and test groups empty

### PR DESCRIPTION
**Pull request by geega**
Had some bug in my own custom command 
When we have included groups as an empty array and the test don't have any groups isGroupAccepted is always false. 
Example
```
count(array_intersect([], [])) === 0) 
```

### CODE REVIEW

File: src/Codeception/Test/Filter.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ public function isNameAccepted(Test $test): bool
 
     public function isGroupAccepted(Test $test, array $groups): bool
     {
-        if ($this->includeGroups !== null && count(\array_intersect($groups, $this->includeGroups)) === 0) {
+        if ($this->includeGroups !== null && $this->includeGroups !== [] && count(\array_intersect($groups, $this->includeGroups)) === 0) {
             return false;
         }
         if ($this->excludeGroups !== null && count(\array_intersect($groups, $this->excludeGroups)) > 0) {
```
Comment (Naktibalda) [inline src/Codeception/Test/Filter.php:102]: Isn't the same patch necessary in this line?
